Tests/Unit/NetworkServer/ChangeTrackingPropertyTest.cs

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace LoRaWan.Tests.Unit.NetworkServerTests
+namespace LoRaWan.Tests.Unit.NetworkServer
 {
-    using LoRaTools.Regions;
+    using global::LoRaTools.Regions;
     using LoRaWan.NetworkServer;
     using Xunit;
 
```

Comment:
comment for a different file: `ConcentratorDeduplicationTest` should also have a namespace without `Test`.

Reply:
Fixed in c46892ab76eb7832c0648cca09cd6148ba74f00b